index.css

```diff
@@ -13,31 +17,32 @@ html {
 }
 
 .container {
-  display: grid;
+  display: flex;
+  flex-direction: column;
   margin: auto;
-  justify-content: start;
-  margin: auto 3em;
+  max-width: 1400px;
+  justify-items: center;
+}
 
-  align-self: center;
-  grid-template-columns: 1fr 3fr 1fr;
-  grid-template-areas:
-    " turn board moves-list"
-    "captured-black board moves-list"
-    "captured-white board moves-list"
-    "btns board  moves-list";
+.capture-container {
+  display: flex;
```

Comment:
One rule of thumb to get used to. `div`  for `display: block`. `span` for `display: inline` or `display-inline-block`. 

assigning `display: inline-block` allows the element to flow inline while being assignable `width` and `height`; `display: inline` will allow inline flow but does not allow for those properties.

using `block`, `inline`, and `inline-block` well will allow you to forego the more heavy handed approaches of `flex` and `grid` for certain simple use cases like this